boundary_layer/schemas/dag.py

```diff
@@ -35,9 +35,15 @@ class ReferenceSchema(OperatorSchema):
     target = fields.String(required=True)
 
 
+class BatchingSchema(StrictSchema):
+    disabled = fields.Boolean()
+    batch_size = fields.Integer(required=True)
+
+
 class GeneratorSchema(ReferenceSchema):
     auto_task_id_mode = fields.String()
     regex_blocklist = fields.List(fields.String())
+    batching = fields.Nested(BatchingSchema)
 
     @validates_schema
     def check_task_id_mode(self, data):
```

Comment:
Oh I did not notice this in the original PR, but we'll have to add logic in this `check_task_id_mode()` method to allow the `auto_task_id_mode` value to be set to `batch_name`, otherwise I think the config parser will reject this setting.  Maybe the logic that you already have for checking these values in `operator.py` could be moved here?